spec/Chassis/Wheel.vspec

```diff
@@ -61,3 +61,13 @@ Tire.Temperature:
   type: sensor
   unit: celsius
   description: Tire temperature in Celsius.
+
+
+#
+# Wheel signals
+#
+Speed:
+  datatype: float
+  type: sensor
+  unit: km/h
```

Comment:
The sensing system measures how fast the wheel rotates. From the sensor point of view, the unit would be something like `radians/second` or `degrees/second`. I believe that in a vehicle, however, this signal is used to derive the vehicle's speed. Since `VSS` does not describe the sensor itself, but rather focuses on the characteristic that the data is supposed to describe, the unit should be the same as the one used for the speed of the vehicle (i.e., `km/h`). 

In the [dataset example](https://github.com/commaai/comma2k19), the unit of this data is measured in `m/s`. Similarly, vendors specify the sensitivity of the sensor in `km/h` (See, for example [this one](https://www.bosch-mobility-solutions.com/en/solutions/sensors/wheel-speed-sensor/) that says "Speed detection close to 0.1 km/h").

Reply:
Then I suggest that the description is changed to just "Speed of a vehicle's wheel", as the term "rotational speed" is misleading (at least, to me).